threads/test_thread_rand.c

```diff
@@ -0,0 +1,88 @@
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/threads.h>
+
+#include "unity_fixture.h"
+
+
+typedef struct {
+	uint16_t time;
+	uint16_t id;
+} threads_self_t;
+
```

Comment:
I don't understand the reason of using types with defined bit size. TIme and id should be defined using common C types e.g. unsigned int.

Reply:
At first i was experimenting with `uint16_t` instead of `char` while defining stack. Changed to unsigned int.